### PR TITLE
Feature: new configuration setting PREVIEW_LMS_BASE

### PR DESCRIPTION
**Pull request by iamCristYe**
discussion here: https://discuss.overhang.io/t/new-settings-variable/1973/4

If my interpretation of https://github.com/overhangio/tutor/blob/926263deb31fab0fe28c1046fd6649e8be31fecd/tutor/config.py#L47 is correct, current users will not need to manually add the new variable for preview domain to work, as missing this config yaml will lead tutor to use the default value.

### CODE REVIEW

File: tutor/interactive.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ def ask_questions(config: Config, defaults: Config) -> None:
         dev_values: Config = {
             "LMS_HOST": "local.overhang.io",
             "CMS_HOST": "studio.local.overhang.io",
+            "PREVIEW_LMS_HOST": "preview.local.overhang.io",
```
Comment (regisb) [inline tutor/interactive.py:44]: I would rather avoid adding an extra question to the quickstart. I think that the default value is reasonable for most people, and we don't need to ask everyone about it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ Note: Breaking changes between versions are indicated by "💥".
 
 ## Unreleased
 
+- [Improvement] Add variable `PREVIEW_LMS_BASE` for custom preview domain.
```
Comment (regisb) [inline CHANGELOG.md:7]: Please replace "variable" by "configuration setting".

**Review by regisb — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (regisb) [reply CHANGELOG.md:7]: Also, this is actually a new feature, so you can replace "Improvement" by "Feature". Please also amend the git commit title accordingly.

**Review by regisb — CHANGES_REQUESTED** [review]

**Comment by regisb** [comment]
Can you please add the discussion link to the git commit message, such that people don't have to research this PR later?

**Comment by iamCristYe** [comment]
@regisb Please check if it's good for merge now 😊

File: tutor/interactive.py [hunk 3]
```diff
@@ -146,16 +155,6 @@ def ask_questions(config: Config, defaults: Config) -> None:
...
-    if run_for_prod:
-        ask_bool(
-            (
-                "Activate SSL/TLS certificates for HTTPS access? Important note:"
-                " this will NOT work in a development environment."
-            ),
-            "ENABLE_HTTPS",
-            config,
-            defaults,
-        )
```
Comment (regisb) [inline tutor/interactive.py:158]: This part of the file should not be modified.

**Review by regisb — CHANGES_REQUESTED** [review]

**Comment by regisb** [comment]
Thanks again for your PR @iamCristYe!

**Pull request merged**